src/Modules/Admin/Commands.Admin/GetPowerBIArtifact.cs

```diff
@@ -0,0 +1,100 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+using Microsoft.PowerBI.Common.Api.Workspaces;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(IEnumerable<Workspace>))]
+    public class GetPowerBIArtifact : PowerBIGetCmdlet, IUserFirstSkip
```

Comment:
Instead of creating a new cmdlet as this just extends the functionality of Get-PowerBIWorkspace. I think we should just add -Expand to Get-PowerBIWorkspace and in Expand it takes an array of enums of Reports, Dashboards, Datasets, etc.

Reply:
I would name the parameter -Include and add an alias for Expand.
Also have an enum value of All to simplify users who want all the artifact types, you could use a flags enum.

Reply:
Updated the PR with suggestion